ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -378,32 +392,80 @@ class ServicesHomeScreen extends React.Component<Props, State> {
 const mapStateToProps = (state: GlobalState) => {
   const { services } = state.entities;
 
+  const potUserMetadata = userMetadataSelector(state);
+  // TODO: disable selection of areas of interest if the user metadata are not noaded
+  // (it causes the new selection is not loaded) https://www.pivotaltracker.com/story/show/168312476
+  const userMetadata = pot.getOrElse(potUserMetadata, noneUserMetadata);
+
   const isAnyServiceLoading =
     Object.keys(services.byId).find(k => {
       const oneService = services.byId[k];
       return oneService !== undefined && pot.isLoading(oneService);
     }) !== undefined;
 
   const isLoading =
-    pot.isLoading(state.entities.services.visible) || isAnyServiceLoading;
+    pot.isLoading(state.entities.services.visible) ||
+    isAnyServiceLoading ||
+    pot.isLoading(potUserMetadata);
 
   return {
     allOrganizations: lexicallyOrderedAllOrganizations(state),
     isLoading,
-    organizationsSelected: organizationsFiscalCodesSelectedStateSelector(state)
+    organizationsSelected: organizationsOfInterestSelector(state),
+    userMetadata
   };
 };
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
   refreshServices: () => dispatch(loadVisibleServices.request()),
-  saveSelectedOrganizationItems: (selectedItemIds: Option<Set<string>>) => {
+  saveSelectedOrganizationItems: (
+    userMetadata: UserMetadata,
+    selectedItemIds: Option<Set<string>>
+  ) => {
     if (selectedItemIds.isSome()) {
-      dispatch(setSelectedOrganizations(Array.from(selectedItemIds.value)));
+      const metadata = userMetadata.metadata;
+      dispatch(
+        userMetadataUpsert.request({
+          ...userMetadata,
+          // tslint:disable-next-line: no-useless-cast
+          version: (userMetadata.version as number) + 1,
+          metadata: {
+            ...metadata,
+            organizationsOfInterest: Array.from(selectedItemIds.value)
+          }
+        })
+      );
     }
   }
 });
 
+const mergeProps = (
+  stateProps: ReturnType<typeof mapStateToProps>,
+  dispatchProps: ReturnType<typeof mapDispatchToProps>,
+  ownProps: OwnProps
+) => {
+  const dispatchUpdateOrganizationsOfInterestMetadata = (
+    selectedItemIds: Option<Set<string>>
+  ) => {
+    if (selectedItemIds.isSome() && stateProps.userMetadata) {
```

Comment:
Could you leave a little description about what is happening?

Reply:
done https://github.com/teamdigitale/io-app/pull/1260/commits/edcbb64642d5c4ac8a8256db8d4ffd0fb43d6226